check_env.py

```diff
@@ -0,0 +1,120 @@
+'''
+Environment checking script for Astropy tutorials:
+This script checks to see if the python environement in the user's computer is safe for running the tutorials.
+The requirements are present in requirements.json files in the respective tutorial folders.
+
+To run this script ,type the following commands from the main astropy-tutorials directory,
+
+$_ python check_env.py -n <tutorial_name>
+
+or
+
+$_ python prepare_deploy.py check -n <tutorial_name>
+
+
+If a specific tutorial name is not provided, the environement will be checked for all tutorials
+
+'''
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+# Standard library
+import os
+import json
+
+try:
+    import astropy
+except ImportError:
+    import logging as logger
+    logger.error("'Astropy' package is missing and is a core dependency to run this script.\
```

Comment:
Both here and below, I would just re-raise an `ImportError` but with your custom message, rather than using `logger.error()`, e.g.,

```python
try:
    import astropy
except ImportError:
    raise ImportError("'Astropy' package is missing and is a core dependency "
                      "to run this script. For more information visit "
                      "http://www.astropy.org/")
```